src/Compilers/CSharp/Portable/LanguageVersion.cs

```diff
@@ -57,14 +57,25 @@ public enum LanguageVersion
         /// </list> 
         /// </remarks> 
         CSharp6 = 6,
+
         /// <summary>
         /// C# language version 7.
         /// </summary>
         CSharp7 = 7,
+
+        /// <summary>
+        /// The latest version of the language supported.
+        /// </summary>
+        Latest = int.MaxValue,
```

Comment:
Given that this enum is an `int`, I've only removed one from the possible set of future versions.

---
In reply to: [72671765](https://github.com/dotnet/roslyn/pull/12796#discussion_r72671765) [](ancestors = 72671765)